lib/registrant.rb

```diff
@@ -0,0 +1,27 @@
+class Registrant
+  attr_reader :name, :age, :permit
```

Comment:
We don't need/want the `:permit` in the attr_reader since we don't want to expose that method. We have the `permit?` acting as a reader.